src/Association/Embedded.php

```diff
@@ -153,6 +154,9 @@ public function getEntityClass(): string
             }
 
             $class = App::className($name, 'Model/Document');
+            if (!$class) {
+                throw new MissingDocumentException([$name]);
+            }
```

Comment:
If the entity class is not set beforehand AND we can't find any in the apps namespace then this is not a valid path to go forward.